esmvalcore/preprocessor/_multimodel.py

```diff
@@ -342,14 +374,14 @@ def _compute_eager(cubes: list, *, operator: iris.analysis.Aggregator,
         result_slices.append(collapsed_slice)
 
     try:
-        result_cube = iris.cube.CubeList(result_slices).concatenate_cube()
+        result_cube = CubeList(result_slices).concatenate_cube()
     except Exception as excinfo:
         raise ValueError(
-            "Multi-model statistics failed to concatenate results into a"
-            f" single array. This happened for operator {operator}"
-            f" with computed statistics {result_slices}."
-            "This can happen e.g. if the calculation results in inconsistent"
-            f" dtypes. Encountered the following exception: {excinfo}")
+            f"Multi-model statistics failed to concatenate results into a "
+            f"single array. This happened for operator {operator} "
+            f"with computed statistics {result_slices}. "
+            f"This can happen e.g. if the calculation results in inconsistent "
+            f"dtypes") from excinfo
```

Comment:
should probably use the lazy % string formatting (even if this f-string was there before)

Reply:
Actually, prospector tells me to use f-strings here, I think that's the more "modern" approach :grimacing:

Reply:
OK am always confused which one's right - Codacy says the lazy string is preferred. Not gonna take any sides :grin: